src/renderer/widgets/IWidget.cpp

```diff
@@ -20,12 +20,18 @@ namespace std {
 }
 #endif
 
+Vector2D rotateVector(const Vector2D& vec, const double& ang) {
+    const double COS = std::abs(std::cos(ang));
+    const double SIN = std::abs(std::sin(ang));
+    return Vector2D(vec.x * COS + vec.y * SIN, vec.x * SIN + vec.y * COS);
+}
+
```

Comment:
We could move that to hyprutils I guess.

Reply:
yes, but for now it has to stay here